css/eu-cookie-compliance.css

```diff
@@ -0,0 +1,34 @@
+.eu-cookie-compliance-save-preferences-button {
+    position: relative;
+}
+
+.eu-cookie-compliance-save-preferences-button.spinning {
+    padding-right: 40px;
+}
+
+.eu-cookie-compliance-save-preferences-button.spinning::before {
+    content: "";
+    position: absolute;
+    width: 0px;
+    height: 0px;
+    border-radius: 50%;
+    right: 6px;
+    top: 50%;
+    box-shadow: 0 0 0 1px rgba(255, 0, 0, 0.1);
```

Comment:
There is a problem here, as the rest of the circle is a 10% opacity version of this colour, but we can't achieve that with CSS variables until CSS values and units 5 is supported (to allow for extracting the r,g,b tuple from the hex and reinserting into an rgba). As such, our CSS currently sets this a 10% red, but this is an artifact of Essex. We can achieve a similar (but aliased) effect by setting it to the colour but 0.1px width, but to my eyes this looks more wrong than the circle being the wrong colour given how faint it is. Personally, I don't think the red is sufficiently visable to be recognisable, but I'd take opinions on this.

Desiredlook (with Croydon colours):
![image](https://user-images.githubusercontent.com/98908/227602006-7b3fc107-d431-4b5e-bb8c-1dfee491fc31.png)

Current:
![image](https://user-images.githubusercontent.com/98908/227602138-e9032d26-0540-481f-b659-6e8d67c9b992.png)

0.1px version:
![image](https://user-images.githubusercontent.com/98908/227602281-0a46665d-ea59-4091-b04a-6e10b4840a88.png)

Any ideas @markconroy? Adding @AnthonySGeorge as this was his change, he may have opinions.

Reply:
Actually, this might work?

`box-shadow: 0 0px 2px 0px var(--button-text-color-hover);`
![image](https://user-images.githubusercontent.com/98908/227603380-0083098e-7680-4a30-ab73-deedeaf2bfbb.png)